src/tests/ftest/util/command_utils.py

```diff
@@ -372,7 +373,8 @@ def set_environment(self, env):
 class CommandWithSubCommand(ExecutableCommand):
     """A class for a command with a sub command."""
 
-    def __init__(self, namespace, command, path="", subprocess=False):
+    def __init__(self, namespace, command, path="", subprocess=False,
```

Comment:
We don't need the `with_json=False` optional argument if the `DaosCommandBase` class is going to define `self.json = FormattedParameter("-j", False)`:
```suggestion
    def __init__(self, namespace, command, path="", subprocess=False):
```

There is some additional code needed at the end of the `__init__` method that will use this argument:
```
        self.json = None
```